pandas/core/indexes/interval.py

```diff
@@ -648,6 +648,8 @@ def _check_method(self, method):
             return
 
         if method in ['bfill', 'backfill', 'pad', 'ffill', 'nearest']:
+            # The documentation warns these methods are not implemented so if
```

Comment:
I don't think this comment actually adds anything here

Reply:
Ok I've dropped it.